### DISCUSSION TITLE
oqsprovider on windows

### DISCUSSION THREAD

**Discussion by spellbam** [open-quantum-safe/oqs-provider#497] [Q&A]
Can anyone tell me whether I built oqsprovider successfully on windows. 

![IMG_7346](https://github.com/user-attachments/assets/264e97ea-5906-4d20-bba6-77de1e2f807e)

**Comment by baentsch** [comment]
Well, it seems to have installed a DLL, so something built... The real test is whether it lists on `openssl list -providers` (assuming OPENSSL_CONF and OPENSSL_MODULES are properly set as per https://github.com/open-quantum-safe/oqs-provider/blob/main/USAGE.md.

**Reply by spellbam** [reply]
i am not able to the oqs provider as mentioned in https://github.com/open-quantum-safe/oqs-provider/blob/main/USAGE.md. and   edit the openssl.conf file as it is residing C:\Program Files\Common Files\SSL
Do you know any way to do this ?

**Reply by baentsch** [reply]
I'm not sure I understand: Are you saying you _cannot_ edit openssl.cnf as per USAGE.md? If so, why?

**Reply by spellbam** [reply]
yes .

**Reply by spellbam** [reply]
![IMG_7347](https://github.com/user-attachments/assets/23aed69a-365e-44d5-988b-52b20177da6b)
It showing like this on my machine 
any way to edit this file ?

**Reply by baentsch** [reply]
The error message is pretty clear: Work with your Windows admin to resolve this; this has nothing to do with OQS. You can use the explicit `-provider` etc. command line options as also documented in USAGE.md.

**Reply by spellbam** [reply]
![image](https://github.com/user-attachments/assets/578fb0e0-8b18-42b4-ad29-22d8a4304649)

It is giving this error ?

**Reply by spellbam** [reply]
![image](https://github.com/user-attachments/assets/d793eafe-de2a-466e-b689-e4e489f5b8a8)

Seems like I have successfully configured oqc provider on my system 🥲. Can anyone verify ?

**Comment by baentsch** [comment]
Looks like the issue is resolved, @spellbam , right? If there's any documentation enhancement to help others on this platform, please contribute it via PR.
